internal/dao/stubs/reservation_dao.go

```diff
@@ -2,14 +2,15 @@ package stubs
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/RHEnVision/provisioning-backend/internal/dao"
 	"github.com/RHEnVision/provisioning-backend/internal/models"
 )
 
 type reservationDaoStub struct {
-	lastId int64
-	store  []*models.AWSReservation
+	lastId   int64
```

Comment:
Oh my god we have it everywhere, it was a copy paste :-)

Ok not here, will fix this.